src/Integration.UnitTests/LocalServices/ProjectSystemFilterTests.cs

```diff
@@ -336,6 +336,29 @@ public void ProjectSystemFilter_SetTestRegex_ArgCheck()
             Exceptions.Expect<ArgumentNullException>(() => testSubject.SetTestRegex(null));
         }
 
+        [TestMethod]
+        public void ProjectSystemFilter_IsAccepted_UnrecognisedProjectType_ReturnsFalse()
```

Comment:
This first test is not related to the fix. I could find any tests that covered non-VB/CS projects so I added this one.